src/java/voldemort/server/protocol/admin/AsyncOperationService.java

```diff
@@ -81,21 +81,28 @@ public synchronized void submitOperation(int requestId, AsyncOperation operation
      * @param requestId Id of the request
      * @return True if request is complete, false otherwise
      */
-    public synchronized boolean isComplete(int requestId) {
-        if(!operations.containsKey(requestId))
-            throw new VoldemortException("No operation with id " + requestId + " found");
+    public synchronized boolean isComplete(int requestId, boolean remove) {
+        //if requestId not found, return true
+        if (!operations.containsKey(requestId))
```

Comment:
Ok well... It still seems a bit unintuitive that any arbitrary requestId that may or many not have ever happened is considered completed. In any case, if we leave it like that, we should at least update the JavaDoc accordingly.